command/operator_debug.go

```diff
@@ -135,6 +140,14 @@ Debug Options:
     The interval between snapshots of the Nomad state. Set interval equal to 
     duration to capture a single snapshot. Defaults to 30s.
 
+  -targets=<metrics,-metrics,...,all,none>
+    Comma separated list of targets to capture.  Defaults to "all".
+    "-<target>" disables capture for that target.
+
+  -interval-targets=<metrics,-metrics,...,all,none>
+    Comma separated list of targets to capture during each interval.  Defaults
+    to "all".  "-<target>" disables capture for that target.
```

Comment:
Do we document which targets are intervals and which ones aren't? Would it make sense to just have a single `-targets` argument without having users worry about the distinction?